modules/cluster-logging-deploy-cli.adoc

```diff
@@ -140,7 +139,7 @@ spec:
   name: "elasticsearch-operator"
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
-<2> Specify `5.0`, `stable`, or `stable-5.<x>` as the channel. See the following note.
+<2> Specify `stable`, or `stable-5.<x>` as the channel. See the following note.
```

Comment:
On 4.8, the available channel are 5.0,stable,stable-5.1,stable-5.2,stable-5.3,stable-5.4